src/js/page/ui/results.js

```diff
@@ -19,30 +19,42 @@ export default class Results {
     this.container = strToEl(
       '<div class="results">' +
         '<span class="size"></span>' +
+        '<span class="spacer"></span>' +
         '<span class="diff"></span>' +
       '</div>' +
     '');
 
     this._sizeEl = this.container.querySelector('.size');
+    this._newSizeEl = this.container.querySelector('.newSize');
     this._diffEl = this.container.querySelector('.diff');
   }
 
   update({ size, comparisonSize }) {
-    this._sizeEl.textContent = humanSize(size);
+    if (comparisonSize) {
+      this._sizeEl.textContent = humanSize(comparisonSize) + ' → ' + humanSize(size);
+    } else {
+      this._sizeEl.textContent = humanSize(size);
+    }
 
     // just displaying a single size?
     if (!comparisonSize) {
       this._diffEl.textContent = '';
       return;
     }
     else if (size == comparisonSize) {
-      this._diffEl.textContent = ' - no change';
+      this._diffEl.textContent = 'no change';
+      this._diffEl.classList.remove('increase');
+      this._diffEl.classList.remove('saving');
     }
     else if (size > comparisonSize) {
-      this._diffEl.textContent = ' - ' + round(size / comparisonSize * 100 - 100, 2) + '% increase';
+      this._diffEl.textContent = '+' + round(size / comparisonSize * 100 - 100, 2) + '%';
+      this._diffEl.classList.add('increase');
+      this._diffEl.classList.remove('saving');
```

Comment:
you could always remove the color classes before the ifs chain, in that way you would save two lines of code 😄

Reply:
:+1: